src/nvim/api/vim.c

```diff
@@ -2453,12 +2453,12 @@ void nvim_del_user_command(String name, Error *err)
   nvim_buf_del_user_command(-1, name, err);
 }
 
-/// Get the command line text.
+/// Get the command line text (same as `getcmdline()`).
```

Comment:
This look good (to address https://github.com/neovim/neovim/pull/18096#discussion_r848977264)? Or should I phrase it differently or something?